dpimport_mriqc.sh

```diff
@@ -24,12 +24,15 @@ fi
 source /data/predict1/utility/.vault/.env.${2}
 
 # remove old data
-mongo --tls --tlsCAFile $state/ssl/ca/cacert.pem --tlsCertificateKeyFile $state/ssl/mongo_client.pem mongodb://dpdash:$MONGO_PASS@$HOST:$PORT/dpdata?authSource=admin --eval "assess=[\"mriqc\",\"eegcount\",\"mricount\",\"mriqcval\"]" /data/predict1/utility/remove_assess.js
+mongo --tls --tlsCAFile $state/ssl/ca/cacert.pem --tlsCertificateKeyFile $state/ssl/mongo_client.pem mongodb://dpdash:$MONGO_PASS@$HOST:$PORT/dpdata?authSource=admin --eval "assess=[\"mriqc\",\"eegcount\",\"mricount\",\"mriqcval\", \"mridataflow\"]" /data/predict1/utility/remove_assess.js
 
 # import new data
 export PATH=/data/predict1/miniconda3/bin:$PATH
 cd ${NDA_ROOT}
 
+# mridataflow checker
+import.py -c $CONFIG "MRI_ROOT/flow_check/*mridataflow-day1to*.csv"
```

Comment:
If it is an individual file, I recommend using `??-*-mridataflow-day1to*.csv` just like we have coded in lines below.

Reply:
I also need to load combined csv files, such as `combined-AMPSCZ*`, `combined-PRONET*` etc. This is why I changed to `*mridataflow-data1to*.csv` pattern.